### ISSUE TITLE
Context Cases Documentation

### ISSUE THREAD

**Issue by noanonoa** [rubocop/rspec-style-guide#136]
I found an issue under [Context Cases](https://github.com/rubocop/rspec-style-guide#context-cases) that goes against the best practices being encouraged in this guide.

```ruby
# bad - needs refactoring
describe '#attributes' do
  context 'the returned hash' do
    it 'includes the display name' do
      # ...
    end

    it 'includes the creation time' do
      # ...
    end
  end
end

# bad - the negative case needs to be tested, but isn't
describe '#attributes' do
  context 'when display name is present' do
    before do
      article.display_name = 'something'
    end

    it 'includes the display name' do
      # ...
    end
  end
end

# good
describe '#attributes' do
  let(:article) { FactoryBot.create(:article) }
  subject(:attributes) { article.attributes }

  context 'when display name is present' do
    before do
      article.display_name = 'something'
    end

    it { is_expected.to include(display_name: article.display_name) }
  end

  context 'when display name is not present' do
    before do
      article.display_name = nil
    end

    it { is_expected.not_to include(:display_name) }
  end
end
```

The `# good` example above is violating the `subject` before `let` principle.  I am enjoying this guide very much and wanted to call out this minor thing.  Thank you for putting this together.

**Comment by andyw8** [comment]
@noanonoa would you like to open a PR to correct this?

**Comment by noanonoa** [comment]
@andyw8 I would love to! Here is the [pull request](https://github.com/rubocop/rspec-style-guide/pull/137)

**Comment by pirj** [comment]
Fixed in #137
